tests/status_response/test_java.py

```diff
@@ -99,3 +101,555 @@ class TestJavaStatusVersion(BaseStatusResponseTest):
     @fixture(scope="class")
     def build(self):
         return JavaStatusVersion.build({"name": "1.8-pre1", "protocol": 44})
+
+
+@BaseStatusResponseTest.construct
+class TestForgeData(BaseStatusResponseTest):
+    EXPECTED_VALUES = [
```

Comment:
Do we really need to test with such a massive response? I don't see any benefit in having more than say 4 channels, and 5 mods. 

I understand that this is probably just the response you got from an actual modpack server, and so it was convenient to test on it, but there's no real benefit in testing the same thing over and over. 

If we can parse 2 channels, we will likely be able to parse 100 channels just as well, no need for the test to actually contain a 100 channels.

Reply:
You are definitely correct with this, this is a real response I had on hand, but it would be nontrivial to change this without an encoder

Reply:
Yeah, I supposed it's fine to just leave it like this. The other option would be to spin up a new forge server, with fewer mods and collect the response from it. But I mean it's not a huge issue to have more mods than necessary here in testing. What is important is to make sure that this really does test every distinct option that the code could encounter though. If it does, I won't insist on making it smaller.

Reply:
If you would be willing to, you can spin up such a server and obtain it's response, if not, feel free to mark this as resolved.